pyrep/robots/mobiles/mobile_base.py

```diff
@@ -27,12 +28,17 @@ def __init__(self,
                  max_velocity: float = 4,
                  max_velocity_rotation: float = 6,
                  max_acceleration: float = 0.035):
-        """Count is used for when we have multiple copies of mobile bases.
+        """Count is used for when we have multiple copies of mobile bases."""
 
-        max_velocity, max_velocityRot, max_acceleration are not used for now for two_wheels robot.
-        distance_from_target is implemented for omnidirectional robot only. It will solve the task
-        by reaching at a distance (distance_from_target) from the target.
         """
+        :param num_wheels: The x, y, z position of the target.
```

Comment:
This doc string doesn't seem right (num_wheels: The x, y, z position of the target (?))
Please check for other cases of this.